src/test/resources/application-serviceAuthPermissions.yaml

```diff
@@ -0,0 +1,11 @@
+s2s:
+  authorizations:
+    payment:
+      id: "payment_app"
+      permission: "R"
+    anotherService:
+      id: "anotherService"
+      permission: "UD"
+    xui:
+      id: "xui_webapp"
+      permission: "CRUD"
```

Comment:
Although these are clearly only test config values: should we use another fake service for "CRUD" rather than giving different permissions than production?  i.e. so it is clear in ServiceAuthorizationConfigTest.java that we expect the service to have all permissions.

Reply:
Yes, good point

Reply:
Yes, good point

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/68/commits/970b0c16e75dc6be385b97fa1c0dd42e08d20d2b